internal/cmd/get/get_test.go

```diff
@@ -71,3 +74,56 @@ func Test_initialiseProjectInCurrentDir(t *testing.T) {
 		})
 	}
 }
+
+func Test_run(t *testing.T) {
+	type args struct {
+		opts *options
+		cmd  *cobra.Command
+	}
+	ctx := context.CLIContext{
+		Config: context.CLIConfig{},
+		Paths: context.Paths{
+			ProjectConfigFile: path.Join(os.TempDir(), "config.yaml"),
+			ProjectInstallDir: os.TempDir(),
+			GlobalInstallDir:  os.TempDir(),
+		},
+	}
+	rootCmdWithGet := cobra.Command{}
+	emptyCmd := cobra.Command{}
+
+	getCmd := NewCommand(ctx)
+	rootCmdWithGet.AddCommand(getCmd)
+
+	tests := []struct {
+		name  string
+		args  args
+		error error
+	}{
+		{
```

Comment:
How about changing the test struct to this?

```go
		{
		       name: "should return error 'The command 'get' is already registered'",
			cmd: &rootCmdWithGet,
			error: fmt.Errorf("The command 'get' is already registered"),
		},
```

that will make the test much more readable. I was thinking about a code like this

```go
	for _, tt := range tests {
		t.Run(tt.name, func(t *testing.T) {
			err := run(ctx, opts, tt.cmd, nil)
			assert.EqualValues(t, err, tt.error)
		})
	}
```

In fact, you don't need to create the `args` struct at all.

Reply:
Same as my comment above.